psutil/_psutil_common.c

```diff
@@ -405,5 +414,9 @@ psutil_setup(void) {
     InitializeCriticalSection(&PSUTIL_CRITICAL_SECTION);
 #endif
 
+#ifdef PSUTIL_OSX
+    mach_timebase_info(&MACH_TIMEBASE_INFO);
+#endif
```

Comment:
I would invoke this from `psutil_setup()` function instead, so that all init functions are in the same place.

Reply:
It is. Do you prefer that I move it before the Windows conditional block?

Reply:
Oh didn't realize you were already in psutil_setup() function, sorry